pycbc/workflow/segment.py

```diff
@@ -1389,3 +1390,58 @@ def file_needs_generating(file_path, cp, tags=None):
             err_msg += '"always", "if_not_present", "error_on_duplicate", '
             err_msg += '"never". Got %s.' %(generate_segment_files,)
             raise ValueError(err_msg)
+
+def get_segments_file(workflow, name, option_name, out_dir):
+    """Get cumulative segments from option name syntax for each ifo.
+
+    Use syntax of configparser string to define the resulting segment_file
+    e.x. option_name = +up_flag1,+up_flag2,+up_flag3,-down_flag1,-down_flag2
+    Each ifo may have a different string and is stored separately in the file.
+
+    Parameters
+    ----------
+    workflow: pycbc.workflow.Workflow
+    name: string
+        Name of the segment list being created
+    option_name: str
+        Name of option in the associated config parser to get the flag list
+
+    returns
+    --------
+    seg_file: pycbc.workflow.SegFile
+        SegFile intance that points to the segment xml file on disk.
+    """
+    from pycbc.events.status import query_combined_flags
+    from pycbc.workflow import SegFile
+    make_analysis_dir(out_dir)
+    cp = workflow.cp
+    start = workflow.analysis_time[0]
+    end = workflow.analysis_time[1]
+
+    # Check for veto definer file
+    veto_definer = None
+    if cp.has_option("workflow-segments", "segments-veto-definer-url"):
+         veto_definer = save_veto_definer(workflow.cp, out_dir, [])
+
+    # Check for provided server
+    server = "segments.ligo.org"
```

Comment:
Should this not default to "https://segments.ligo.org, as it used to?

Reply:
I can't test this, but I think it should be without the url header given that the query command itself has a separate input for this part.